### PR TITLE
[Enhancement]: Privacy Screen layout improvement.

### PR DESCRIPTION
**Pull request by utkarsh006**
## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

## Screenshots / Screen Recordings
Please attach screenshots if you have changed the UI, and screen recordings to demonstrate that no existing functionality is broken and that the issue is resolved.

### CODE REVIEW

**Comment by Vishwajith-Shettigar** [comment]
@utkarsh006  please remove unnecessary formatting changes.

**Comment by utkarsh006** [comment]
@Vishwajith-Shettigar I don't think they are unnecessary, they are just adhering the basic guidelines.

File: app/src/main/java/com/example/nextgen/privacy/OnPrivacyItemClicked.kt [hunk 1]
```diff
@@ -4,5 +4,5 @@ import com.example.model.Privacy
 import com.example.model.PrivacyItem
 
 interface OnPrivacyItemClicked {
-  fun onPrivacyItemClicked(privacyItem: PrivacyItem,status:Boolean,index: Int)
+    fun onPrivacyItemClicked(privacyItem: PrivacyItem, status: Boolean, index: Int)
```
Comment (Vishwajith-Shettigar) [inline app/src/main/java/com/example/nextgen/privacy/OnPrivacyItemClicked.kt:7]: please remove such formatting changes.

File: app/src/main/java/com/example/nextgen/privacy/PrivacyActivity.kt [hunk 2]
```diff
@@ -13,29 +13,29 @@ import com.example.utility.getProtoExtra
...
-    super.onCreate(savedInstanceState)
-    setContentView(R.layout.activity_privacy)
-    val profile =
-      intent.getProtoExtra(PRIVACYACTIVITY_INTENT_EXTRAS_KEY, Profile.getDefaultInstance())!!
-    supportFragmentManager.beginTransaction()
-      .replace(R.id.frame_layout, PrivacyFragment.newInstance(profile))
-      .commit()
-  }
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
```
Comment (Vishwajith-Shettigar) [inline app/src/main/java/com/example/nextgen/privacy/PrivacyActivity.kt:17]: same in this files

File: app/src/main/res/layout/privacy_items_layout.xml [hunk 3]
```diff
@@ -3,49 +3,59 @@
...
-            android:gravity="center_vertical|end" />
-    </LinearLayout>
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+    </androidx.constraintlayout.widget.ConstraintLayout>
 
-</layout>
+</layout>
```
Comment (Vishwajith-Shettigar) [inline app/src/main/res/layout/privacy_items_layout.xml:61]: Please add extra one line in EOF.

File: app/src/main/res/layout/fragment_privacy.xml [hunk 4]
```diff
@@ -24,8 +28,12 @@
...
+            android:layout_height="0dp"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/appbar_layout" />
 
-    </LinearLayout>
-</layout>
+    </androidx.constraintlayout.widget.ConstraintLayout>
+</layout>
```
Comment (Vishwajith-Shettigar) [inline app/src/main/res/layout/fragment_privacy.xml:39]: same here EOF extra one line.

File: app/src/main/java/com/example/nextgen/privacy/RouteToPrivacyActivity.kt [hunk 5]
```diff
@@ -3,5 +3,5 @@ package com.example.nextgen.privacy
 import com.example.model.Profile
 
 interface RouteToPrivacyActivity {
-  fun routeToPrivacyActivity(profile: Profile)
+    fun routeToPrivacyActivity(profile: Profile)
```
Comment (Vishwajith-Shettigar) [inline app/src/main/java/com/example/nextgen/privacy/RouteToPrivacyActivity.kt:6]: remove space.

File: app/src/main/java/com/example/nextgen/privacy/PrivacyViewModel.kt [hunk 6]
```diff
@@ -14,77 +15,78 @@ import com.example.nextgen.viewmodel.ObservableViewModel
...
+        viewModelScope.launch {
+            profileController.updatedisableProfilePicture(userId = profile.userId, choice) {
+                if (it is com.example.utility.Result.Success) {
+                    callback(com.example.utility.Result.Success("Successful"))
+                } else {
 
-          callback(com.example.utility.Result.Failure("Failed"))
+                    callback(com.example.utility.Result.Failure("Failed"))
+                }
+            }
```
Comment (Vishwajith-Shettigar) [inline app/src/main/java/com/example/nextgen/privacy/PrivacyViewModel.kt:87]: same here

File: app/src/main/java/com/example/nextgen/privacy/PrivacyViewModel.kt [hunk 7]
```diff
@@ -14,77 +15,78 @@ import com.example.nextgen.viewmodel.ObservableViewModel
...
-  init {
-    loadPrivacyItems()
+    init {
+        loadPrivacyItems()
 
-  }
+    }
 
-  fun loadPrivacyItems() {
+    fun loadPrivacyItems() {
```
Comment (Vishwajith-Shettigar) [inline app/src/main/java/com/example/nextgen/privacy/PrivacyViewModel.kt:32]: same here

File: app/src/main/java/com/example/nextgen/privacy/PrivacyFragment.kt [hunk 8]
```diff
@@ -1,202 +1,186 @@
...
+            arguments = Bundle().apply {
+                putProto(PRIVACY_FRAGMENT_ARGUMENTS_KEY, profile)
+            }
         }
-      }
-    } else {
     }
-  }
-}
+}
```
Comment (Vishwajith-Shettigar) [inline app/src/main/java/com/example/nextgen/privacy/PrivacyFragment.kt:186]: EOF add extra line.

File: app/src/main/java/com/example/nextgen/privacy/PrivacyFragment.kt [hunk 9]
```diff
@@ -1,202 +1,186 @@
...
+    override fun injectDependencies(fragmentComponent: FragmentComponent) {
+        fragmentComponent.inject(this)
     }
 
-    binding.lifecycleOwner = this
-
-    privacyViewModel.privacyitemsList.observe(viewLifecycleOwner) {
-      privacyAdapter.itemList = it as MutableList<PrivacyItemsViewModel>
-    }
+    override fun onCreateView(
```
Comment (Vishwajith-Shettigar) [inline app/src/main/java/com/example/nextgen/privacy/PrivacyFragment.kt:50]: please only change your code.

**Comment by Vishwajith-Shettigar** [comment]
Please only add code if you are improving any feature or adding new feature. Don't push unnecessary code changes.

**Pull request closed without merging**